ingest-app/src/cmr/ingest/api/routes.clj

```diff
@@ -82,6 +84,21 @@
            {ctx :request-context params :params}
            (acl/verify-ingest-management-permission ctx :update)
            (subscriptions-helper/trigger-email-subscription-processing ctx params)
+           {:status 200})
+     (GET  "/email-subscription-processing-job-state"
+           {ctx :request-context}
+           (acl/verify-ingest-management-permission ctx :update)
```

Comment:
Shouldn't need extra permission to check the processing state.

Reply:
Removed the permission for this endpoint